@app/db/scripts/dump-db.js

```diff
@@ -0,0 +1,31 @@
+const { spawn } = require("child_process");
+
+const inTests = process.env.IN_TESTS;
+if (!inTests) {
```

Comment:
Let's exit early instead, so we don't have to look for an `else` block at the end of the file when reading the code.

```suggestion
if (process.env.IN_TESTS === "1") {
  process.exit(0);
}
```